core/tests/ledger_cleanup.rs

```diff
@@ -170,14 +174,22 @@ mod tests {
     /// - `STOP_SIZE_BYTES`: if specified, the benchmark will count how
     ///   many times the ledger store size exceeds the specified threshold.
     /// - `STOP_SIZE_ITERATIONS`: when `STOP_SIZE_BYTES` is specified, the
-    ///   benchmark will stop immediately when the number of times where the
-    ///   ledger store size exceeds the configured `STOP_SIZE_BYTES`.  These
-    ///   configs are used to make sure the benchmark runs successfully under
-    ///   the storage limitation.
+    ///   benchmark will stop immediately when the number of consecutive times
+    ///   where the ledger store size exceeds the configured `STOP_SIZE_BYTES`.
+    ///   These configs are used to make sure the benchmark runs successfully
+    ///   under the storage limitation.
```

Comment:
nit:
"when `STOP_SIZE_BYTES` is specified, this value specifies how many consecutive times the ledger storage size can exceed `STOP_SIZE_BYTES` before the benchmark is terminated.
This option can be used to ensure the benchmark runs without exceeding a storage limitation."